### PR TITLE
Aruha 905

### PR DESCRIPTION
**Pull request by zrobgar**
Create a Basic Security.md file for nakadi

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Allows open access to *Nakadi*. This mode only exists for testing purposes.
+
+## Token Validation
+
+Validates the Token, and only the token
+
+## Token and Scope Validation
+
+Validates the scope of the token as well as the existence of the token.
```
Comment (lmontrieux) [inline SECURITY.md]: s/scope/scopes (there can be >1)

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+*Nakadi* makes use of Oauth2 to provide security, there are three modes that can be 
+configured. In order to configure security for *Nakadi* you need to edit the *Nakadi* configuration file and add oauth2
+ validation endpoints. This is set via two environmental variables.
+ 
+ * AuthenticationMode: Security model to use 
+ 
+ * TokenInfoUrl: oauth2 endpoint to get token information
+
+## OpenAuthenticationMode
```
Comment (lmontrieux) [inline SECURITY.md]: Maybe add the value here (NONE)? Same remark for the next 2 sections, BASIC and FULL

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Allows open access to *Nakadi*. This mode only exists for testing purposes.
```
Comment (adyach) [inline SECURITY.md]: The possible values are https://github.com/zalando/nakadi/blob/master/src/main/java/org/zalando/nakadi/config/SecuritySettings.java#L12

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Security Model
+
+
+*Nakadi* makes use of Oauth2 to provide security, there are three modes that can be 
+configured. In order to configure security for *Nakadi* you need to edit the *Nakadi* configuration file and add oauth2
```
Comment (adyach) [inline SECURITY.md]: application.yml (env vars) VS *Nakadi* configuration file

Comment (zrobgar) [inline SECURITY.md]: thanks done

File: SECURITY.md (hunk 3 above)
Comment (zrobgar) [inline SECURITY.md]: Thanks, corrected

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Security Model
+
+
+*Nakadi* makes use of Oauth2 to provide security, there are three modes that can be 
+configured. In order to configure security for *Nakadi* you need to edit the *Nakadi* application.yml (env vars)
+                                                                                      
+and add oauth2
```
Comment (adyach) [inline SECURITY.md]: formatting broken

**Comment by adyach** [comment]
👍

**Comment by zrobgar** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Pull request merged**